src/js/index.js

```diff
@@ -14,8 +14,7 @@ const sort = {
 const contentEl = document.getElementById('content');
 const filterInput = document.getElementById('filter');
 const sortingInput = document.getElementById('sorting');
-const color_switch = document.getElementById('color-switch');
-const body = document.body;
+const colorSwitch = document.getElementById('color-switch');
```

Comment:
Please add a newline after your variable (it's the last one of the list)

Reply:
Got it.